analyzer/src/main/kotlin/managers/Pub.kt

```diff
@@ -594,9 +594,27 @@ class Pub(
     private fun installDependencies(workingDir: File) {
         requireLockfile(workingDir) { workingDir.resolve(PUB_LOCK_FILE).isFile }
 
-        // The "get" command creates a "pubspec.lock" file (if not yet present) except for projects without any
-        // dependencies, see https://dart.dev/tools/pub/cmd/pub-get.
-        run(workingDir, "get")
+        if (containsFlutter(workingDir)) {
```

Comment:
Commit message:
- s/flutter/Flutter/ (at least if you mean the tool not the CLI command; if you do mean the CLI command, put it in backticks)
- No comma after "flutter"
- Add a comma after "SCM"